velox/functions/prestosql/types/tests/CMakeLists.txt

```diff
@@ -12,10 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+add_library(velox_presto_types_test_lib TypeTestBase.cpp)
+target_link_libraries(
+  velox_presto_types_test_lib GTest::gtest)
```

Comment:
Not sure what the purpose is of having this extra lib? why not just conditionally add the geometrytypetest.cpp?

Reply:
Ah, sorry I should have caught that... :see_no_evil: 

> Just as a side thought, I wonder if we could add some documentation 

Yes, I have been neglecting that, sorry.

> I do have a question about the velox_link_libraries implementation though. 

Hm, yes seems you are correct. I added that with the initial PR where it was necessary but it probably was refactored away at some point. I don't know the exact target it was used on anymore ^^